src/components/Auths/authenticationButton.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { useAuth0 } from '@auth0/auth0-react';
+import LoginButton from './loginButton';
+import LogoutButton from './logoutButton';
+
+function AuthenticationButton() {
+  const { isAuthenticated } = useAuth0();
+
+  return isAuthenticated ? <LogoutButton /> : <LoginButton />;
```

Comment:
Very trivial at this scale, but consider using `if` statements instead of ternary operations as they're more readable.